browser/css/jquery-ui-lightness.css

```diff
@@ -1108,9 +1104,6 @@ a.ui-button:active,
 .ui-state-error-text .ui-icon {
 	background-image: url("images/ui-icons_ffd27a_256x240.png");
 }
-.ui-button .ui-icon {
```

Comment:
No need for different background-images. One background-image (black/white) and the other color stuff was done with  hover effect.